src/index.js

```diff
@@ -135,15 +137,16 @@ export default class Carousel extends Component {
 
   componentDidUpdate(prevProps, prevState) {
     const { children, autoplay, slideWidth } = this.props;
-    const { currentSlide, loadedImages, direction, loading, slideDimensions } = this.state;
+    const { currentSlide, loadedImages, direction, loading, slideDimensions, slideAlignment } = this.state;
```

Comment:
This should be coming from `this.props` not from `this.state`

Reply:
Fixed, thanks!